### ISSUE TITLE
Does this work?

### ISSUE THREAD

**Issue by fbchrt** [django-wong/launchbar-giphy#1]
Hello,

was very excited when I found that LB Action, but unfortunately the preview CMD+Y (maybe the most important part?) doesn't work for me: I'm only seeing a static default file icon in the preview window.

Is it just me or is this a bug?

**Comment by django-wong** [comment]
I haven't seen such an issue on three of my mac computers. Hard to tell its just one case or bug.
Did you see thumbnails on the left side of the list?

<img src="https://user-images.githubusercontent.com/4019240/87634151-4ddf5280-c76f-11ea-9b98-f17601565959.png" width="300px">

**Comment by fbchrt** [comment]
Yes, I see (static) thumbnails.

<img width="660" alt="Bildschirmfoto 2020-07-16 um 21 57 51" src="https://user-images.githubusercontent.com/5232478/87716795-a7677180-c7af-11ea-8b15-0a5b496d6b88.png">

**Comment by django-wong** [comment]
The thumbnails and quick look preview are referring to the same file on the disk, can you try...
- Open 'Finder' and Go to Folder '~/Library/Caches/at.obdev.LaunchBar/Actions/com.djangowong.LaunchBar.action.GIPHY/images'
- Find a gif file and press <kbd>space</kbd> see if QuickLook works there?

**Comment by fbchrt** [comment]
Yes, QuickLook works fine there ... lots of gifs in that folder!

**Comment by youtextme** [comment]
For LaunchBar → Raycast buy/switch: keep LaunchBar on ⌘Space, map free Raycast to ⌥Space for a week, and only pay for Raycast Pro (or stay on LaunchBar) after clipboard/file actions actually stick in daily use.

https://raycast.com/?via=girish
Affiliate — I earn a commission if you upgrade to Pro.
